spec/lib/jobs_spec.rb

```diff
@@ -5,7 +5,7 @@
     subject(:search) do
       Jobs.search({ query:'Nursing jobs',
                     organization_codes: 'HE38',
-                    location_name: 'Washington, DC, USA',
+                    location_name: 'Baltimore, MD, USA',
```

Comment:
Why the change from DC to Baltimore? This doesn't affect the test.

Reply:
This tests NIH search results to come back with results even if we are in Baltimore.